docs/FreeBSD.md

```diff
@@ -0,0 +1,55 @@
+# FreeBSD Install
```

Comment:
keep file names only lower case.

the first header should match the file name. (The header can be capitalized as wished)

Reply:
https://www.azerothcore.org/wiki/wiki-standards